src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -7,6 +7,7 @@
       outlined
       color="primary"
       :ripple="false"
+      tabindex="-1"
```

Comment:
Why do we want to remove this from the tab order?

Reply:
Same question for the two other buttons in the other files.

Reply:
I removed the `v-expansion-panel` from tab order first but the buttons were still were in the tab order. See below

![image](https://github.com/bcgov/business-filings-ui/assets/116035339/e2063629-bdb4-42d1-9ad5-301dda8cba63)


https://github.com/bcgov/business-filings-ui/assets/116035339/a872973b-7fef-41c0-8615-aae4cc90ff6f

Reply:
This is an extension of "hide the collapse button for incomplete IAs/registrations/etc" change, right?

I think the view button(s) should still be visible but the hide buttons should not (nor be tabbable).

Reply:
Using the preview link above, I can still tab to the panel header and press Space or Enter to toggle it. That's what we want to get rid of.

tabbed to panel header
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/bfe2e2d1-198d-4ce7-8282-3cd82af65e83)

Reply:
I used the same link but I can't tab to the panel. This is weird!

https://github.com/bcgov/business-filings-ui/assets/116035339/765bb436-1f01-48a2-b15c-65326a413356

Reply:
Click the View button to expand the panel, then try tabbing (or shift-tabbing).

Reply:
This code change was reverted so this comment is obsolete. I'm resolving it.